network-ntt/src/client.rs

```diff
@@ -241,35 +288,26 @@ impl<B: Block + HasHeader> ConnectionState<B> {
 
 enum Command<B: Block + HasHeader, Tx: TransactionId> {
     Message(Message<B, Tx>),
-    BlockHeaders(
-        LightWeightConnectionId,
-        Response<BlockHeaders<B::Header>, String>,
-    ),
-    Blocks(LightWeightConnectionId, Response<B, String>),
-    Transaction(LightWeightConnectionId, Response<bool, String>),
     Request(Request<B>),
     CloseConnection(LightWeightConnectionId),
+    Reply(LightWeightConnectionId, Vec<u8>),
 }
 
-enum V<A1, A2, A3, A4, A5, A6, A7> {
+enum V<A1, A2, A3, A4, A5> {
     A1(A1),
     A2(A2),
     A3(A3),
     A4(A4),
     A5(A5),
-    A6(A6),
-    A7(A7),
 }
 
-impl<A1, A2, A3, A4, A5, A6, A7> Future for V<A1, A2, A3, A4, A5, A6, A7>
+impl<A1, A2, A3, A4, A5> Future for V<A1, A2, A3, A4, A5>
```

Comment:
This horror should be removed by my state machine rewrite, but let it linger for a the time being.

Reply:
Afaik the commit that introduced the change didn't compile yet? Otherwise I will happily rebase on that.